### DISCUSSION TITLE
Hello, does ContentLayer support loading MDX text from remote API?

### DISCUSSION THREAD

**Discussion by ghost** [contentlayerdev/contentlayer#479] [Q&A]
Hello, does ContentLayer support loading MDX text from remote API?

**Comment by RamiroG8k** [comment]
Hi, it looks like there's a new section that says it is as experimental feature, their example is using `github`:

https://www.contentlayer.dev/docs/sources/remote-files-fbb47906

Maybe you should check it out!, Let us know what happens 😄
